src/components/AddressSearch.js

```diff
@@ -36,51 +35,44 @@ const defaultProps = {
 };
 
 const AddressSearch = (props) => {
-    const googlePlacesRef = useRef();
-    useEffect(() => {
-        if (!googlePlacesRef.current) {
-            return;
-        }
-
-        googlePlacesRef.current.setAddressText(props.value);
-    }, []);
+    const [skippedFirstOnChangeText, setSkippedFirstOnChangeText] = useState(false);
```

Comment:
Can you add a comment to explain what this is doing? I haven't seen this before and it isn't used anywhere else in the codebase

Reply:
Oh, that is how you keep states in functional components. I think it isn't anywhere in the code because when there is a state, we use class components. In this case, there was some discussion about difficulties in converting this component into a class component.. will look for that.

If this stays like this, I'll add the comment you are requesting.

Reply:
Ah this is the context cc @Luke9389 https://expensify.slack.com/archives/C03TQ48KC/p1634088400387400

Reply:
But if we're removing the ref and `useEffect()` then this should be converted back into a class component maybe 😂

Reply:
I can give it a try to converting it to class component now then.

Reply:
I tried converting it to a class component and had issues using local state with the library. We could definitely give it a try, but watch out for that.